homeassistant/components/statistics/sensor.py

```diff
@@ -248,11 +245,8 @@ def valid_boundary_configuration(config: dict[str, Any]) -> dict[str, Any]:
         vol.Optional(CONF_SAMPLES_MAX_BUFFER_SIZE): vol.Coerce(int),
         vol.Optional(CONF_MAX_AGE): cv.time_period,
         vol.Optional(CONF_PRECISION, default=DEFAULT_PRECISION): vol.Coerce(int),
-        vol.Optional(
-            CONF_QUANTILE_INTERVALS, default=DEFAULT_QUANTILE_INTERVALS
-        ): vol.All(vol.Coerce(int), vol.Range(min=2)),
-        vol.Optional(CONF_QUANTILE_METHOD, default=DEFAULT_QUANTILE_METHOD): vol.In(
-            ["exclusive", "inclusive"]
+        vol.Optional(CONF_PERCENTILE, default=50): vol.All(
```

Comment:
This is a breaking change and not a deprecation, right?

Reply:
Correct.